File: MerlinRestNasir/src/main/java/merlin/revature/beans/UserPublicInfo.java
package merlin.revature.beans;

public class UserPublicInfo {
	
	private int userId;
	private String username, fname, lname;

	public UserPublicInfo(int userId, String username, String fname, String lname) {
		this.userId = userId;
		this.username = username;
		this.fname = fname;
		this.lname = lname;
	}
	
	public int getUserId() {
		return userId;
	}
	
	public void setUserId(int userId) {
		this.userId = userId;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getFname() {
		return fname;
	}

	public void setFname(String fname) {
		this.fname = fname;
	}

	public String getLname() {
		return lname;
	}

	public void setLname(String lname) {
		this.lname = lname;
	}
	
	

}

File: Merlin/src/app/components/models/user-public-info.ts
export class UserPublicInfo {
    
    fname: string;
    lname: string;
    userId: number;
    username: string;

    constructor(fname: string, lname: string, userId: number, username: string) {
        this.fname = fname;
        this.lname = lname;
        this.userId = userId;
        this.username = username;
    }
}
